recipe/meta.yaml

```diff
@@ -39,7 +39,7 @@ test:
     - test
   requires:
     - pytest
-    - shapely  # [not osx or py<=39]
+    - shapely
```

Comment:
Shapely should now be available for all builds we want to do, except that CI failed on one particular one (`linux linux_aarch64_numpy1.19python3.9.____73_pypy`) and I just restarted it.